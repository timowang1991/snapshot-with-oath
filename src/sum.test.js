// file: sum.test.js

import sum from './sum';

it('should sum 2 numbers', () => {
    const result = sum(1, 2);

    expect(result).toBe(3);
});

// file: sum.test.js
it('should sum 2 numbers snapshot', () => {
    expect(sum(1, 2)).toMatchSnapshot();
});