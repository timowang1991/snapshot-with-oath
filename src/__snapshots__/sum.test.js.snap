// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should sum 2 numbers snapshot 1`] = `3`;
